tapir/coop/apps.py

```diff
@@ -73,9 +73,13 @@ def register_emails():
         from tapir.coop.emails.co_purchaser_updated_mail import (
             CoPurchaserUpdatedMail,
         )
+        from tapir.accounts.emails.create_account_reminder_email import (
+            CreateAccountReminderEmail,
+        )
```

Comment:
`CreateAccountReminderEmail` is in `tapir.accounts`, so maybe it shouldn't be called here, but I couldn't make it work differently (there is no `apps.py` in `accounts`).

Reply:
It's not too bad since `accounts` and `coop` depend on each other anyway, but do you remember what the error was when you added an `apps.py` to `accounts`?

Reply:
Looks like this was some other error I had. Now it works, so will move the code to `tapir/accounts/apps.py`.